### PR TITLE
A VERDADE

### PR DESCRIPTION
**Pull request by Vinimatayoshi**
apenas aceit3e

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <title>Curso git</title>
 </head>
 <body>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi similique eos unde ex dolores tempora corrupti molestiae dolorum in accusantium iure voluptatum, vel consectetur cupiditate assumenda. Dolorum deleniti hic ex?</p>
+    <p>O Omar é Vasco</p>
     <img src="assets/svg/icon_trophy.svg" alt="">
```
Comment (Omar-Vinicius) [inline index.html:11]: NAO NAO E POSSIVEL ISSO NAAAAO

File: index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <title>Curso git</title>
 </head>
 <body>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi similique eos unde ex dolores tempora corrupti molestiae dolorum in accusantium iure voluptatum, vel consectetur cupiditate assumenda. Dolorum deleniti hic ex?</p>
+    <p>O Omar é Vasco</p>
```
Comment (Omar-Vinicius) [inline index.html:10]: AIIIII

File: index.html (hunk 1 above)
Comment (Vinimatayoshi) [reply index.html:11]: :}

Comment (Omar-Vinicius) [reply index.html:11]: OOOOOOOPPPPPPPPAAAAAAAAAAAAAA

**Pull request merged**